package.json

```diff
@@ -40,7 +40,7 @@
     "@types/jest": "^23.3.11",
     "@types/prop-types": "^15.5.8",
     "@types/react": "^16.7.18",
-    "@types/react-modal": "^3.6.0",
+    "@types/react-modal": "3.8.0",
```

Comment:
I had to force the version because I don't know why but it always downloaded a newer version which breaks the modal package.